src/main/java/seedu/address/logic/candidate/EditCandidateCommand.java

```diff
@@ -59,7 +59,10 @@
 
     public static final String MESSAGE_EDIT_PERSON_SUCCESS = "Edited Candidate: %1$s";
     public static final String MESSAGE_NOT_EDITED = "At least one field to edit must be provided.";
-    public static final String MESSAGE_DUPLICATE_PERSON = "This candidate already exists in the address book.";
+    public static final String MESSAGE_DUPLICATE_PERSON = "Candidate with Email:"
+            + " [ %1$s ] already exists in the HR Manager";
+    public static final String MESSAGE_ILLEGAL_PERSON_STATUS = "Unable to change Status of %1$s to APPLIED.\n"
+            + "Candidate %1$s already have scheduled interview(s).";
```

Comment:
Sorry, didnt see this earlier. Candidate has instead of have

Reply:
oops my bad, changing it now!